cpp/libcudf_kafka/include/cudf_kafka/kafka_consumer.hpp

```diff
@@ -100,9 +100,9 @@ class kafka_consumer : public cudf::io::datasource {
   std::string buffer;
 
  private:
-  RdKafka::ErrorCode update_consumer_toppar_assignment(std::string const &topic,
-                                                       int partition,
-                                                       int64_t offset);
+  RdKafka::ErrorCode update_consumer_topic_partition_assignment(std::string const &topic,
```

Comment:
Oh, I thought you weren't going to change this (since it's a standard kafka term). I'm fine with either.